lib/tasks/subscriber_list_audit.rake

```diff
@@ -0,0 +1,48 @@
+require "sidekiq/api"
+require "sitemap-parser"
+
+namespace :subscriber_list_audit do
+  desc "Create audit workers"
+  task :start, %i[batch_size limit] => :environment do |_t, args|
+    batch_size = args.fetch(:batch_size, "100").to_i
+    limit = args.fetch(:limit, "-1").to_i
+
+    puts("Creating audit workers with batch size #{batch_size}, limit #{limit}")
+
+    base_sitemap = SitemapParser.new("#{Plek.website_root}/sitemap.xml")
+    sitemap_parts = base_sitemap.sitemap.at("sitemapindex").element_children.map { |element| element.at("loc").content }
+
+    sitemap_parts = limit == -1 ? sitemap_parts : sitemap_parts.take(1)
```

Comment:
Ah, this is for short-term testing. Because each section of the sitemap has 25,000 urls, generally if you wanted to just run this with the first 100 content items, that meant you only wanted the first sitemap - I think I'll get rid of this, because I'm confident it works now.